diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt

```diff
@@ -61,4 +61,17 @@ class TrivialPropertyAccessorsWarnTest : LintTestBase(::TrivialPropertyAccessors
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(TRIVIAL_ACCESSORS_ARE_NOT_RECOMMENDED)
+    fun `should not trigger on private setter`() {
```

Comment:
Please also add case with empty getter: they are useless but technically allowed in the language.
```kotlin
val x = 0
    get
```